src/endWindow.java

```diff
@@ -15,6 +15,7 @@
      */
     public endWindow() {
         initComponents();
+int a=0;
           jLabel5.setText("User Has Wins :");
 ;          jLabel3.setText("Computer Has Wins :");
             Integer val=new Integer(ticTak12.db.result("user"));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this "Integer" constructor [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS2129)